deploy/install.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+set -o errexit
+
+echo "" > appsmith_deploy.log
+
+declare -A osInfo;
+
+osInfo[/etc/debian_version]="apt-get"
+osInfo[/etc/centos-release]="yum"
+osInfo[/etc/redhat-release]="yum"
+
+# Checking OS and assiging package manager
+desired_os=0
+echo "Assigning package manager"
+for f in ${!osInfo[@]}
+do
+    if [[ -f $f ]];then
+        package_manager=${osInfo[$f]}
+        echo $package_manager
+        desired_os=1
+    fi
+done
+
+if [[ $desired_os -eq 0 ]];then
+    echo "This script is currently meant to install Appsmith on Ubuntu | RHEL | CentOS machines."
+    echo "Please contact hello@appsmith.com with your OS details if you wish to extend this support"
+    echo "Exiting for now. Bye!"
+	exit
+fi
+
+read -p 'Installation Directory [appsmith]: ' install_dir
+install_dir=${install_dir:-appsmith}
+mkdir -p $PWD/$install_dir
+install_dir=$PWD/$install_dir
+echo "Appsmith needs a mongodb instance to run"
+echo "1) Automatically setup mongo db on this instance (recommended)"
+echo "2) Connect to an external mongo db"
+read -p 'Enter option number [1]: ' mongo_option
+mongo_option=${mongo_option:-1}
+if [[ $mongo_option -eq 2 ]];then
+    read -p 'Enter your mongo db host: ' mongo_host
+    read -p 'Enter the mongo root user: ' mongo_root_user
+    read -sp 'Enter the mongo password: ' mongo_root_password
+    read -p 'Enter your mongo database name: ' mongo_database
+elif [[ $mongo_option -eq 1 ]];then
+    mongo_host="mongo"
+    mongo_database="appsmith"
+    read -p 'Set the mongo root user: ' mongo_root_user
+	read -sp 'Set the mongo password: ' mongo_root_password
+fi
+echo ""
+read -p 'Would you like to setup a custom domain to access appsmith? [Y/n]: ' setup_domain
+setup_domain=${setup_domain:-Y}
+
+if [ $setup_domain == "Y" -o $setup_domain == "y" -o $setup_domain == "yes" -o $setup_domain == "Yes" ];then
+	read -p 'Enter your domain name (example.com): ' custom_domain
+fi
+
+NGINX_SSL_CMNT=""
+if [ ! $custom_domain ];then
```

Comment:
Please revisit this condition. We are running the command inside `$custom_domain`, which I doubt is the intention here. We are probably checking whether the `$custom_domain` variable's value is empty.

```suggestion
if [[ -n $custom_domain ]]; then
```

Reply:
Done. Thanks for spotting this.